content/applications/finance/accounting/reporting/customize.rst

```diff
@@ -160,6 +160,16 @@ expression's **label** (ex. **code.label**).
    The value of the arithmetic expression will be returned only if it is strictly between the
    provided bounds. Otherwise, it will be brought back to the closest bound.
 
+`if_other_expr_above(LINE_CODE.EXPRESSION_LABEL, CUR(amount))`
+   The value of the arithmetic expression will be returned only if the value of the expression denoted
```

Comment:
```suggestion
   The value of the arithmetic expression will be returned only if the value of the expression denoted
```
It seems "denoted" is two characters past the character limit. Your technical knowledge is much better than mine, so I'm not sure if there's a specific reason or if it's just a mistake :)

I'll approve it, but if it is indeed a mistake, could you please fix it?